resources/js/features/games/components/GameAchievementSetsContainer/SetSelectionControl/SetSelectionControl.tsx

```diff
@@ -15,9 +16,19 @@ export const SetSelectionControl: FC<SetSelectionControlProps> = ({ activeTab })
 
   // On mobile with 5+ sets, use a dropdown for better UX.
   if (ziggy.device === 'mobile' && selectableGameAchievementSets.length >= 5) {
-    return <SetSelectionDropdown activeTab={activeTab} />;
+    return (
+      <div className="flex w-full items-center gap-2">
+        <SetSelectionDropdown activeTab={activeTab} />
+        <SubsetConfigurationButton />
+      </div>
+    );
   }
 
   // Otherwise, use the tab interface.
-  return <SetSelectionTabs activeTab={activeTab} />;
+  return (
+    <div className="flex w-full justify-between gap-2">
+      <SetSelectionTabs activeTab={activeTab} />
```

Comment:
This seems to be broken. If I refresh a page, I can switch to/from the subset, but then I can't switch back until I refresh again.

Reply:
I still see this behavior.

https://github.com/user-attachments/assets/59e52e28-77b5-4c4c-90d1-b089fbe7ad1a